dist/routable-component.js

```diff
@@ -134,7 +134,7 @@ var routableComponent =
 	     * @param {Object<RoutableComponent>} components コンポーネントのリスト
 	     * @return {void}
 	     */
-	    value: function register_components(components) {
+	    value: function registerComponents(components) {
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)
* Warning - This function has too many statements (15). Maximum allowed is 10. (max-statements)
* Warning - Function 'registerComponents' has a complexity of 5. (complexity)

<!-- Comment by [SideCI](https://sideci.com) -->